doc/distrib/NodeHelpFiles/en-US/RUAGD4YGKJ6XHPIKDL7GZX63CDAFMI6KUSR6XMXEBGJJOATEI5IA.md

```diff
@@ -1,7 +1,6 @@
-<!--- Autodesk.DesignScript.Geometry.TSpline.TSplineSurface.BySphereBestFit --->
-<!--- RUAGD4YGKJ6XHPIKDL7GZX63CDAFMI6KUSR6XMXEBGJJOATEI5IA --->
-## BySphereBestFit - Documentation
```

Comment:
oh my bad here, @zeusongit I dont think we want to remove the comments since they indicate the original file name

Reply:
I only found this out when looking at the corresponding localization PR

Reply:
Ok, so do we want to retain the first 2 lines here?

Reply:
Yes @zeusongit Please, this is so that we can locate and update easier for these md files